org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -1026,6 +1049,11 @@ public String getBlobDataType() {
 	public String getClobDataType() {
 		return "CLOB";
 	}
+	
+	@Override
+	public String getJsonDataType() {
+		return "JSON";

```

Comment:
this require Oracle 21c and above.  before 21c, it is usually store as varchar or clob and use the check constraint (columnName is JSON).